### PR TITLE
Feature/update forms

### PR DESCRIPTION
**Pull request by ShrawanBK**

### CODE REVIEW

File: src/components/EventForm/index.tsx [hunk 1]
```diff
@@ -596,11 +609,22 @@ function EventForm(props: EventFormProps) {
             </div>
             {!readOnly && (
                 <div className={styles.actions}>
+                    {!!onHideEventModal && (
```
Comment (tnagorra) [inline src/components/EventForm/index.tsx:612]: I think we don't need this check and always pass onHideEventModal

File: src/components/EventForm/index.tsx [hunk 2]
```diff
@@ -228,6 +231,7 @@ function EventForm(props: EventFormProps) {
         id,
         crisisId,
         readOnly,
+        onHideEventModal,
```
Comment (tnagorra) [inline src/components/EventForm/index.tsx:234]: The proper prop name should be `onFormCancel`

File: src/views/Crises/index.tsx [hunk 3]
```diff
@@ -346,6 +346,7 @@ function Crises(props: CrisesProps) {
                         <CrisisForm
                             id={crisisIdToEdit}
                             onCrisisCreate={handleCrisisCreate}
+                            onHideAddCrisisModal={closeAddCrisisModal}
```
Comment (tnagorra) [inline src/views/Crises/index.tsx:349]: The proper name would be onFormCancel

File: src/components/EventForm/index.tsx (hunk 1 above)
Comment (ShrawanBK) [reply src/components/EventForm/index.tsx:612]: Event form is not always called inside a modal (Referring to code in **EntryForm**). So, onHideEventModal is optional and could be undefined.

Comment (tnagorra) [reply src/components/EventForm/index.tsx]: OOh

**Review by tnagorra — APPROVED** [review]

**Comment by tnagorra** [comment]
@ShrawanBK can you rebase this pull request?

**Comment by ShrawanBK** [comment]
> @ShrawanBK can you rebase this pull request?

**Review by tnagorra — APPROVED** [review]

**Pull request merged**